log.py

```diff
@@ -264,6 +324,12 @@ class LogConfiguration(object):
         },
     ]
 
+    @classmethod
+    def _defaults(cls, testing=False):
+        log_level = cls.DEFAULT_LOG_LEVEL
+        database_log_level = cls.DEFAULT_DATABASE_LOG_LEVEL
+        return log_level, database_log_level
+
```

Comment:
I think you can get rid of this method and just use the constants. I spent a while confused why there were two different methods called `_defaults` and we don't need this one at all.

Reply:
Sure. I had that method to make it easier to test the defaults, but I'll pull it out.